dracut/repartition/endless-repartition.sh

```diff
@@ -203,7 +203,7 @@ fi
 # remove the last-lba line so that we fill the disk
 # + Randomize PARTUUIDs
 # + Randomize GPT disk UUID
-parts=$(echo "$parts" | sed -e '/^last-lba:/d;s/ uuid=.*//;/^label-id:/d')
+parts=$(echo "$parts" | sed -e '/^last-lba:/d;s/ uuid=[a-fA-F0-9-]\{36\},\{0,1\}//;/^label-id:/d')
```

Comment:
can you remind me what the \{0,1\} bit does here?

Can you include in the commit message the line of sfdisk output showing the UUID followed by partition attributes? so that we have a clear documented example of input data that we are working with here..

Reply:
> can you remind me what the {0,1} bit does here?

After the UUID can be present `{1}` or not `{0}` a `,` depending on the number of attributes.

I'll fix the commit messages.

Reply:
Thanks that is clearer.

But search for "Remove marker" further down in this file. Is that code not also problematic for you? I guess it also removes the LegacyBIOSBootable flag.

Reply:
That was addressed in the second commit if I'm identifying the point correctly.